packages/suite/src/utils/wallet/coinControlUtils.ts

```diff
@@ -0,0 +1,31 @@
+import { AccountOutputLabels } from '@suite-common/metadata-types';
+import { Utxo } from '@trezor/blockchain-link';
+
+/**
+ * Filter UTXOs based on search query and categorize them into spendable, low anonymity and dust UTXOs.
+ */
+export const filterAndCategorizeUtxos = (
+    searchQuery: string,
+    utxos: Utxo[],
+    spendableUtxos: Utxo[],
+    lowAnonymityUtxos: Utxo[],
+    dustUtxos: Utxo[],
+    outputLabels: { [txid: string]: AccountOutputLabels }
```

Comment:
I think, we are using the pattern that parameters are always nested inside the object so we can have named params.

In this case:

```ts
type FilterAndCategorizeUtxos = {
    searchQuery: string,
    utxos: Utxo[],
    spendableUtxos: Utxo[],
    lowAnonymityUtxos: Utxo[],
    dustUtxos: Utxo[],
    outputLabels: { [txid: string]: AccountOutputLabels }
}

export const filterAndCategorizeUtxos = ({...}: FilterAndCategorizeUtxos) => ....
```